ethtool_test.go

```diff
@@ -165,3 +165,24 @@ func TestFeatures(t *testing.T) {
 		t.Fatalf("loopback interface reported all features available")
 	}
 }
+
+func TestIdentity(t *testing.T) {
```

Comment:
it seems that this test can't pass on the CI. You could use `if testing.Short()` to skip it in the CI.

Reply:
done